src/__tests__/core/web_api/p2_api.test.js

```diff
@@ -79,6 +83,14 @@ describe('Auth0APIClient', () => {
         const client = getClient({ params: { state: 'foo' } });
         expect(client.authOpt.state).toBe('foo');
       });
```

Comment:
what if both `state` are set, which one should win? Add that test and document this behavior somewhere.

Reply:
added